website/content/en/docs/component-guides/kserve.md

```diff
@@ -8,6 +8,8 @@ This tutorial shows how to set up a load balancer endpoint for serving predictio
 
 > Note: Kubeflow on AWS v1.4 uses [KFServing](https://www.kubeflow.org/docs/external-add-ons/kserve/kserve/#kfserving-is-now-kservehttpskservegithubiowebsite07blogarticles2021-09-27-kfserving-transition). The KFServing project is now called KServe.
```

Comment:
lets remove both the lines or maybe just add one line about KFServing project is now called KServe wiht this link https://kserve.github.io/website/0.7/blog/articles/2021-09-27-kfserving-transition/#kfserving-is-now-kserve and link to this guide for migration https://kserve.github.io/website/0.7/admin/migration/